superset-frontend/src/dashboard/components/PropertiesModal.tsx

```diff
@@ -103,8 +84,38 @@ const loadOwnerOptions = (input = '') => {
   );
 };
 
-class PropertiesModal extends React.PureComponent {
-  constructor(props) {
+type PropertiesModalProps = {
+  dashboardId: number;
+  show: boolean;
+  onHide: () => void;
+  colorScheme?: string;
+  setColorSchemeAndUnsavedChanges: () => void;
```

Comment:
This isn't anything you changed, but I don't see `setColorSchemeAndUnsavedChanges` anywhere else in this file. Think it can be removed entirely?